locales/pt-BR/translation.json

```
@@ -96,6 +96,43 @@
   "$FixedDayCalendar": {
     "leave-by": "Você deve sair às:"
   },
+  "$WorkdayWaiver": {
+    "add-waiver": "Adicionar renúncia",
+    "already-have-waiver": "Você já tem uma renúncia em",
+    "attention": "Atenção: A terceirização de feriados em dias com isenção anulará a isenção.",
+    "city": "Cidade:",
+    "conflicts": "Conflitos?",
+    "country": "País:",
+    "date": "Encontro",
+    "day": "Dia",
+    "delete": "Excluir",
+    "delete-waiver-message": "Tem certeza de que deseja excluir a renúncia no dia", 
+    "end-date": "Data final",
+    "end-date-cannot-be-less": "A data de término não pode ser inferior à data de início.",
+    "from": "A partir de",
+    "header-help": "As alterações entram em vigor ao fechar esta janela",    
+    "holiday": "Feriado",
+    "hours": "Horas",
+    "hours-waived": "Horas dispensadas",
+    "import": "Importar?",
+    "loaded-waivers-holidays": "Isenções carregadas para feriados.",
+    "nav-holiday": "Feriado",
+    "no-working-days-on-range": "Não é possível adicionar renúncia. A faixa não contém nenhum dia útil.",
+    "on-working-day": "Em dia de trabalho",
+    "reason": "Razão",
+    "remove-waiver": "Remova-o antes de adicionar um novo.",
+    "source-selected-holidays": "Férias selecionadas de origem",
+    "start-date": "Data de início",
+    "state": "Estado:",
+    "time-to-leave": "Hora de partir",
```

Time to Leave is the app name, we are not translating it:

```suggestion
    "time-to-leave": "Time to Leave",
```